src/database.py

```diff
@@ -24,22 +26,54 @@
 
 metadata = MetaData(naming_convention=DB_NAMING_CONVENTION)
 
+
+language = Table(
+    "language",
+    metadata,
+    Column("code", String, primary_key=True),
+    Column("emoji", String, nullable=False),
```

Comment:
Добавить колонну с тем, показываем ли мы этот язык пользователям
```suggestion
    Column("emoji", String, nullable=False),
    Column("is_active", Boolean, nullable=False, default=True),
```